ansible/templates/common/introduction.j2

```diff
@@ -1,3 +1,3 @@
 {% if doc %}
-This is an example walkthrough for installing OMERO.web in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables.
-{% endif %}
\ No newline at end of file
+This is an example walkthrough for installing OMERO.web in a **virtual environment** using a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough, we will use the **{{ omero_user }} system user** and define the main OMERO.web configuration options as environment variables. Since 5.6, a new :envvar:`OMERODIR` variable is used, you should first unset :envvar:`OMERO_HOME` (if set) before beginning the installation process.
```

Comment:
NB: can this long line be wrapped?

Reply:
The display when generated is not ideal. I tried in the past and did not get the desired outcome
I can try gain